### ISSUE TITLE
Unable to start two depthcache for pairs

### ISSUE THREAD

**Issue by duduzhu** [sammchardy/python-binance#18]
I created client and did this:
```
bnbbtc_dcm = DepthCacheManager(client, 'BNBBTC', display_any_msg)
bnbeth_dcm = DepthCacheManager(client, 'BNBETH', display_any_msg)
```
And I got error like this. It seems that simultaneously creating multiple depthcachemanager is not possible?

```
Exception in thread Thread-2:
Traceback (most recent call last):
  File "C:\Python27\lib\threading.py", line 801, in __bootstrap_inner
    self.run()
  File "C:\Python27\lib\site-packages\binance\websockets.py", line 305, in run
    reactor.run(installSignalHandlers=False)
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 1242, in run
    self.startRunning(installSignalHandlers=installSignalHandlers)
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 1222, in startRunning
    ReactorBase.startRunning(self)
  File "C:\Python27\lib\site-packages\twisted\internet\base.py", line 728, in startRunning
    raise error.ReactorAlreadyRunning()
ReactorAlreadyRunning
```

**Comment by sammchardy** [comment]
@duduzhu I've released v0.3.5 which suppresses this error and allows you to run multiple Depth caches.

**Comment by duduzhu** [comment]
It's ok to create multiple depthcaches now, but I find they share the same cache now...
Following is my code, each time these 3 depthcaches give the same result.
```
def process_any_depth(any_depth_cache):
    global bnbbtc_dcm, bnbeth_dcm, ethbtc_dcm
    bnbeth_dc = bnbeth_dcm.get_depth_cache()
    bnbbtc_dc = bnbbtc_dcm.get_depth_cache()
    ethbtc_dc = ethbtc_dcm.get_depth_cache()
    
    print("BNBETH:{}".format(bnbeth_dc.get_bids()[0]))
    print("BNBBTC:{}".format(bnbbtc_dc.get_bids()[0]))
    print("ETHBTC:{}".format(ethbtc_dc.get_bids()[0]))
    


bnbbtc_dcm = DepthCacheManager(client, 'BNBBTC', process_any_depth)
bnbeth_dcm = DepthCacheManager(client, 'BNBETH', process_any_depth)
ethbtc_dcm = DepthCacheManager(client, 'ETHBTC', process_any_depth)
```

**Comment by duduzhu** [comment]
Following is what I got by depthcach.get_bids()
`[[0.048451, 9.859], [0.04845, 12.0], [0.048449, 0.278], [0.04843, 15.545], [0.048411, 0.001], [0.048408, 12.683], [0.048407, 12.683], [0.048383, 0.41], [0.04834, 0.095], [0.048336, 2.804], [0.048316, 7.7], [0.047757, 1.506], [0.00480002, 1250.0], [0.0048, 908.0], [0.0047952, 18.0], [0.00474188, 88.0], [0.00474045, 18.0], [0.00474044, 182.0], [0.00474043, 6.0], [0.00473, 80.0], [0.00472201, 1265.0], [0.004722, 218.0], [0.00351215, 1062.0], [0.00023111, 331.0], [0.00023107, 5.0], [0.00023106, 6.0], [0.000231, 1250.0], [0.00023025, 70.0], [0.00023001, 341.0], [0.00023, 178.0], [0.00022917, 144.0], [0.00022908, 10.0], [0.00022801, 4385.0], [0.00022001, 700.0]]`

**Comment by duduzhu** [comment]
I suggest this.
```
class DepthCache(object):

    def __init__(self, symbol):
        """Intialise the DepthCache

        :param symbol: Symbol to create depth cache for
        :type symbol: string

        """
        self._symbol = symbol
        self._bids = {}
        self._asks = {}
```

**Comment by sammchardy** [comment]
@duduzhu yes you're right it does seem to be sharing the same cache.
I will have to look into updating to allow separation in some way like your suggestion.

**Comment by sammchardy** [comment]
@duduzhu I've released v0.3.7 with your fix above. Can confirm separate caches as expected.
Thanks for your help.

**Comment by duduzhu** [comment]
@sammchardy you are welcome. Thanks for your fast improvement.
